packages/app-bridge/src/utilities/MessageQueue.spec.ts

```diff
@@ -1,14 +1,14 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { describe, expect, it, vi } from 'vitest';
-import { MessageBus } from './MessageBus';
+import { MessageQueue } from './MessageQueue';
 
 describe('MessageBus', () => {
     it('should be instantiable', () => {
         const { port1 } = new MessageChannel();
 
-        const messageBus = new MessageBus(port1);
-        expect(messageBus).toBeInstanceOf(MessageBus);
+        const messageBus = new MessageQueue(port1);
```

Comment:
Might as well rename the local variables here also to `messageQueue` for consistency